book/translation.rst

```diff
@@ -502,6 +502,11 @@ as the locale for the current request.
 You can now use the locale to create routes to other translated pages
 in your application.
 
+.. tip::
+
+    Read :doc:`/cookbook/routing/service_container_parameters` to learn, how to
+    avoid hardcoding the ``_locale`` requirement in all your routes.
```

Comment:
This is exactly about route requirements as in the example route above, isn't it?